doc/man-sections/script-options.rst

```diff
@@ -586,6 +586,24 @@ instances.
     netsh.exe calls which sometimes just do not work right with interface
     names). Set prior to ``--up`` or ``--down`` script execution.
 
+:code:`dns_*`
+    The ``--dns`` configuration options will be made available to script
+    execution through this set of environment variables. For the semantics
+    of each individual variable, please refer to the documentation for ``--dns``.
+
+    ::
+
+       dns_search_domain_{n}
+       dns_server_{n}_address4
+       dns_server_{n}_port4
```

Comment:
Now that this can be unset in the case where user didn't specify a port, I think this behavior should be explicitely documented. I.e. add something like:

```
If no port was specified, ``dns_server_{n}_port4`` and / or ``dns_server_{n}_port6`` will be undefined. This usually means that the value should be guessed based on ``dns_server_{n}_transport``.
```

Reply:
It is the same for resolve/exclude_domain and search_domain. Now even wondering if we need dnssec=no and transport=plain, as they are too going to be defaulting to that in the backend likely.

Reply:
Added a general remark, now I'm going to add a `unset` value for dnssec and transport and display them only when set.